rdrf/rdrf/management/commands/sync_proms.py

```diff
@@ -16,7 +16,8 @@ def add_arguments(self, parser):
                             help='To overwrite proms system metadata with site system metadata. Skip this option to preserve metadata.')
 
     def handle(self, *args, **options):
+        override = options.get("override")
         for registry_model in Registry.objects.all():
             if registry_model.proms_system_url:
```

Comment:
PromsProcessor now is a bad name

better

PromsSystemManager

instead of sync_proms maybe update_definition (  management command should be django-admin update-proms-system-definition

pull_proms is ok

Reply:
The code just needs to make it obvious that in one case we are updating the **definition** to  the proms site, the pull proms is getting the **patient data** from the proms site.

Reply:
review changes made